paddle/math/tests/CMakeLists.txt

```diff
@@ -14,3 +14,4 @@ add_simple_unittest(test_perturbation)
 add_simple_unittest(test_CpuGpuVector)
 add_simple_unittest(test_Allocator)
 add_simple_unittest(test_FPException)
+add_simple_unittest(test_GpuProfiler)
```

Comment:
please update to the newest develop code, and use the `pre commit` hooks. This file has a bad eof